lib/musicxml/node/base.rb

```diff
@@ -23,7 +23,12 @@ def parse_nodes(node)
           self.class.stored_nodes.each do |name|
             klass = find_class(symbol_to_class(name))
             node_list = node.search(symbol_to_node(name)).map do |child_node|
-              klass.new(child_node)
+              begin
+                klass.new(child_node)
+              rescue => exception
+                puts "Could not create node of type #{klass}: #{exception.message}"
```

Comment:
No not strictly necessary, but useful for debugging as it gives a more meaningful error message than a full trace.  Perhaps we should re-throw after printing the class and node that are behaving naughty?